bfvmm/include/hve/arch/intel_x64/save_state.h

```diff
@@ -50,8 +50,8 @@ struct save_state_t {
 
     uint64_t reserved1;             // 0x088
     uint64_t reserved2;             // 0x090
-    uint64_t vcpu_ptr;              // 0x098
-    uint64_t exit_handler_ptr;      // 0x0A0
+    uint64_t vcpuid;                // 0x098
+    uint64_t vcpu_ptr;              // 0x0A0
```

Comment:
I don't think I messed something up here, I deleted the exit handler class, and changed to exit handler assembly entry point to go to a vcpu member function.